eng/common/scripts/Package-Properties.ps1

```diff
@@ -162,6 +162,14 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
     $allPackageProperties = Get-AllPkgProperties
     $diff = Get-Content $InputDiffJson | ConvertFrom-Json
     $targetedFiles = $diff.ChangedFiles
+
+    if ($diff.DeletedFiles) {
```

Comment:
Will this work? If we have deleted files in the list of changed files won't we try to build projects that don't exist?

Reply:
I am relying on `save-package-properties` NOT generating a project for that deleted folder. If the package itself is deleted, then none of those files will map back to a package.

![image](https://github.com/user-attachments/assets/06d29939-b0a5-4d1f-a625-3dbe34582496)